obsreport/obsreport.go

```diff
@@ -62,15 +67,19 @@ func setParentLink(parentCtx context.Context, childSpan *trace.Span) bool {
 
 // Configure is used to control the settings that will be used by the obsreport
 // package.
-func Configure(level configtelemetry.Level) (views []*view.View) {
+func Configure(level configtelemetry.Level) *ObsMetrics {
```

Comment:
Probably this should be moved in internal if possible since this API will no longer exists when switching to otel metrics.

Reply:
Yes, made the changes and updated the PR description.